src/Google.Api.Gax/ServiceSettingsBase.cs

```diff
@@ -44,6 +58,7 @@ public abstract class ServiceSettingsBase
         {
             settings.CallSettings = CallSettings.Clone();
             settings.Clock = Clock;
+            settings.UserAgent = UserAgent;
```

Comment:
Thinking about this more, we could remove CloneInto but add two constructors: one parameterless one, and one that accepts a ServiceSettingsBase to clone from.